src/pages/place/[place_id]/index.tsx

```diff
@@ -0,0 +1,152 @@
+import { useRouter } from "next/router";
+import useSWR from "swr";
+import Link from "next/link";
+
+const linkClass =
+  "text-black border-b-black border-b-2 hover:pb-2 hover:border-b-black";
+
+const fetcher = (...args: Parameters<typeof fetch>) =>
+  fetch(...args).then(res => res.json());
+
+const fieldsToNames = {
+  guideDogAvg: "Guide Dog Friendliness",
+  isMenuAccessibleAvg: "Menu Accessibility",
+  noiseLevelAvg: "Noise Level",
+  lightingAvg: "Lighting",
+  isStaffHelpfulAvg: "Staff Helpfulness",
+  isBathroomOnEntranceFloorAvg: "Bathroom On Entrance Floor",
+  isContactlessPaymentOfferedAvg: "Contactless Payment Offered",
+  isStairsRequiredAvg: "Stairs Required",
+  spacingAvg: "Spacing",
+};
+
+const validRatingFields = Object.keys(fieldsToNames);
+
+const getColorFromFromFillPct = (fillPct: number) => {
+  if (fillPct < 25) {
+    return "bg-red-600";
+  } else if (fillPct < 50) {
+    return "bg-orange-400";
+  } else if (fillPct < 75) {
+    return "bg-yellow-500";
+  } else {
+    return "bg-green-700";
+  }
+};
+
+const PlacePage = (): JSX.Element | null => {
+  const { query } = useRouter();
+  const { data, error } = useSWR<
+    { placeDetails: { results: any; accessibilityData: any } }, // TODO change to correct types when packag is available
```

Comment:
Do we wanna wait until the package is ready before we merge this in?

Reply:
nah

Reply:
typescript is mostly useful during development, so if we're adding things that should ADD new types then we would block them from merge, but i wouldn't block something that is just going to USE types if the dev work is already done

Reply:
gotcha!